starknet_py/net/full_node_client.py

```diff
@@ -101,6 +104,79 @@ async def get_block_traces(
     ) -> BlockTransactionTraces:
         raise NotImplementedError()
 
+    async def get_events(
+        self,
+        *,
+        address: Hash,
+        keys: List[str],
+        from_block_number: Optional[Union[int, Tag]] = None,
+        from_block_hash: Optional[Union[Hash, Tag]] = None,
+        to_block_number: Optional[Union[int, Tag]] = None,
+        to_block_hash: Optional[Union[Hash, Tag]] = None,
+        follow_continuation_token: bool = False,
+        chunk_size: int = 1,
+    ) -> EventsResponse:
```

Comment:
How do you make a request with a continuation token returned from this method? You can `follow_continuation_token` automatically but what I one wants to manually follow returned token.

Reply:
Fixed!